### PR TITLE
Full width for audio UI

### LINKED ISSUE #2875: Display a wider audio
I've been using this excellent software for years (thank you very much), but now I started following a podcast. For me it works good enough, but I'd like the audio element to be wider, because right now it's difficult to skip parts of the audio.

### PR DESCRIPTION
**Pull request by Alkarex**
#fix https://github.com/FreshRSS/FreshRSS/issues/2875

### CODE REVIEW

**Comment by Alkarex** [comment]
image

**Comment by Elinvention** [comment]
:+1:  tested and that's exactly what I asked for!

**Comment by Elinvention** [comment]
ありがとうございます。Arigatō gozaimasu. :smile_cat:

**Review by aledeg — APPROVED** [review]

File: p/themes/base-theme/template.css [hunk 1]
```diff
@@ -93,10 +93,14 @@ img.favicon {
 }
 
 /*=== Videos */
-iframe, embed, object, video {
+audio, iframe, embed, object, video {
 	max-width: 100%;
 }
 
+audio {
+	width: 100%;
```
Comment (Frenzie) [inline p/themes/base-theme/template.css:101]: Doesn't this override the max-width above? (can't check atm)

Comment (Alkarex) [reply p/themes/base-theme/template.css:101]: `max-width` wins over `width`. I am not sure we actually need the `max-width` rule, but it was mainly to be consistent with the rules used for `video` etc.

**Review by Frenzie — APPROVED** [review]

**Comment by Alkarex** [comment]
@Elinvention Merged in the /master branch, and you are welcome to help testing that branch before the release of version 1.16.0 in the coming days

**Pull request merged**